api/src/main/scala/hmda/api/processing/lar/SingleLarValidation.scala

```diff
@@ -0,0 +1,59 @@
+package hmda.api.processing.lar
+
+import akka.actor.{ Actor, ActorLogging, Props }
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.engine.ValidationError
+import hmda.validation.engine.lar.LarEngine
+
+object SingleLarValidation {
+  def props: Props = Props(new SingleLarValidation)
+
+  case class CheckLar(lar: LoanApplicationRegister)
+  case class CheckSyntacticalLar(lar: LoanApplicationRegister)
+  case class CheckValidityLar(lar: LoanApplicationRegister)
```

Comment:
I'd expect these to be called something like `CheckLarValidity` (etc).  Or, since they're already in the context of an object with `Lar` in its name, then perhaps even just `CheckValidity`, `CheckSyntactical`, and `Check` (or `CheckEverything`). Is there a reason it's helpful to keep repeating the `Lar` part?